src/Event/Subscriber/ContentSaveSubscriber.php

```diff
@@ -39,10 +39,34 @@ public function postSave(ContentEvent $event): ContentEvent
         return $event;
     }
 
+    public function preDelete(ContentEvent $event): ContentEvent
+    {
+        // Saving an entry in the log now. post_delete doesn't have content anymore.
+        $context = [
+            'content_id' => $event->getContent()->getId(),
+            'content_type' => $event->getContent()->getContentType(),
+            'title' => $event->getContent()->getExtras()['title'],
+        ];
+
+        $this->logger->info('Deleted content "{title}" ({content_type} № {content_id})', $context);
+
+        return $event;
+    }
+
+    public function postDelete(ContentEvent $event): ContentEvent
+    {
+        // Make sure we flush the cache for the menu's
```

Comment:
This is a plain plural, not a possessive. ;-)

```suggestion
        // Make sure we flush the cache for the menus
```

Reply:
🙈